model/Security/Classes/VulnAssessmentRelationship.md

```diff
@@ -0,0 +1,48 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# VexVulnAssessmentRelationship
+
+## Summary
+
+Asbtract ancestor class for all VEX relationships
+
+## Description
+
+VexVulnAssessmentRelationship is an abstract subclass that defined the common
+properties shared by all the SPDX-VEX status relationships. 
+
+## Constraints
+
+When linking elements using a VexVulnAssessmentRelationship, the following
+requirements must be observed:
+
+- The from: end must be a /Security/Vulnerability classed element
+- The to: end must point to elements representing the VEX _products_. To
+specify a different element where the vulnerability was detected, the VEX
+relationship can optionally specify _subcomponents_ using the`assessedElements
```

Comment:
missing closing ` character

Reply:
Mmh this file is wrong. It seems to be copy pasta from the vex ancestor class.